src/python/elements.cpp

```diff
@@ -216,6 +216,20 @@ void init_elements(py::module& m)
         )
     ;
 
+    py::class_<elements::Aperture>(mx, "Aperture")
+        .def(py::init<>(),
+             "Mixin class for lattice elements with a transverse aperture."
+        )
```

Comment:
I know we have it above for the other mixin classes, but looking at it we might want to check if we can remove this constructor for all of them, so one cannot create these mixin classes as a standalone variable, which would cause confusion.
```suggestion
```

Reply:
Removed now for all mixin classes.